logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/scenario/OnClientsRequest.kt

```diff
@@ -0,0 +1,59 @@
+package com.wire.kalium.logic.feature.call.scenario
+
+import com.sun.jna.Pointer
+import com.wire.kalium.calling.Calling
+import com.wire.kalium.calling.callbacks.ClientsRequestHandler
+import com.wire.kalium.calling.types.Handle
+import com.wire.kalium.logic.callingLogger
+import com.wire.kalium.logic.data.call.CallClient
+import com.wire.kalium.logic.data.conversation.ConversationRepository
+import com.wire.kalium.logic.data.id.toConversationId
+import com.wire.kalium.logic.functional.map
+import com.wire.kalium.logic.functional.onFailure
+import com.wire.kalium.logic.functional.onSuccess
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+class OnClientsRequest(
```

Comment:
What do you think of marking this class as `internal` ? I believe classes that are only used within the scope of kalium (and in `:logic` module) should be restricted to the outside clients =D